### ISSUE TITLE
Unable to uninstall

### ISSUE THREAD

**Issue by beechow** [adobe/brackets#14883]
unable to uninstall.  please see screen

version: 1.6
os: win10

[image](https://user-images.githubusercontent.com/12259851/66271786-e23d9000-e894-11e9-92d6-c64607e7b1b5.png)

**Comment by mhornbacher** [comment]
This looks like an windows registry configuration issue, did you remove the files from disk manually before attempting to uninstall?

**Comment by beechow** [comment]
it seems not looking for the program files.  it seems looking for the .msi installer which i have deleted long time ago after installations... 
why it needs to look for the installer?  any workaround?

**Comment by mhornbacher** [comment]
Windows only recently introduced the concept of the OS uninstalling an application (and almost no one uses this feature).

In windows the responsibility is on the developer to provide an exe that will uninstall their application. When you "uninstall" an application it just runs that file. Many companies are famous for not fully removing their software from your machine.

Are you sure it is looking for the installer (in downloads) or is it looking for an MSI file in program files?

**Comment by beechow** [comment]
Hi it looks for 
Brackets.1.6.Extract.msi
The file name appears to be an installer which usually with .msi extension and the word extract is a typical early installation step to extract installation files.

**Comment by mhornbacher** [comment]
Yes but in what location?

**Comment by beechow** [comment]
it's asking the .msi from my user download folder.  this one is probably where I downloaded Brackets 1.6 installer before.

**Comment by beechow** [comment]
no. install new version request me to uninstall the 1.6 leading to this problem.
and it's difficult to find 1.6 installer again.

and i've given up.  turned out resetting the whole windows.
